metricbeat/module/jolokia/jmx/_meta/docs.asciidoc

```diff
@@ -1,58 +1,67 @@
-mapping:
+For example, to get the `Uptime` attribute from the `java.lang:type=Runtime`
+MBean and map it to an event field called `jolokia.jmx.uptime`, configure the
+following mapping:
 
 [source,yaml]
----
+----
 - module: jolokia
   metricsets: ["jmx"]
   hosts: ["localhost:8778"]
-  namespace: "metrics"
+  namespace: "metrics" <1>
   jmx.mappings:
     - mbean: 'java.lang:type=Runtime'
       attributes:
         - attr: Uptime
-          field: uptime
-          event: uptime
+          field: uptime <2>
+          event: uptime <3>
       target:
           url: "service:jmx:rmi:///jndi/rmi://targethost:9999/jmxrmi"
           user: "jolokia"
           password: "s!cr!t"
----
+----
+<1> The `namespace` setting is required.
+<2> This field will be called `jolokia.jmx.uptime` in the output event because
+the Beats framework adds a prefix to the name.  
+<3> The `event` setting is optional. Use this setting to group all attributes
+with the same `event` value into the same event when sending data to Elastic. 
 
-In case the underlying attribute is an object (e.g. see HeapMemoryUsage attribute in java.lang:type=Memory) its
-structure will be published to Elastic "as is".
+If the underlying attribute is an object (such as the `HeapMemoryUsage`
+attribute in `java.lang:type=Memory`), its structure will be published to
+Elastic "as is".
 
-It is possible to configure nested metric aliases by using dots in the mapping name (e.g. gc.cms_collection_time). For examples please refer to the
+You can configure nested metric aliases by using dots in the mapping name (for
+example, `gc.cms_collection_time`). For more examples, see
 https://github.com/elastic/beats/blob/{doc-branch}/metricbeat/module/jolokia/jmx/_meta/test/config.yml[/jolokia/jmx/test/config.yml].
 
-All metrics from a single mapping will be POSTed to the defined host/port and sent to Elastic as a single event.
-To make it possible to differentiate between metrics from multiple similar applications running on the same host,
-please configure multiple modules.
+All metrics from a single mapping will be POSTed to the defined host/port and
+sent to Elastic as a single event. To make it possible to differentiate between
+metrics from multiple similar applications running on the same host, you should
+configure multiple modules.
 
-When wildcards are used, an event will be sent to Elastic for each matching mbean, in that case a `mbean` field is added.
-
-Optionally, an `event` name can be added to each attribute, this makes all metrics with the same `event`
-to be grouped in the same event when being sent to Elastic.
-
-It is required to set a namespace in the general module config section.
+When wildcards are used, an event is sent to Elastic for each matching
+MBean, and an `mbean` field is added to the event.
 
 [float]
 === Limitations
-No authentication against Jolokia is supported yet.
```

Comment:
The truth is that I haven't tested that, but indeed it should work as it uses the same http helper as other modules.